### ISSUE TITLE
Low accuracy

### ISSUE THREAD

**Issue by nnt296** [luckycallor/InsightFace-tensorflow#3]
@luckycallor, thanks for your code!
However, I tried to evaluate your model **[config_ms1m_100_334k](https://pan.baidu.com/s/1Zr91ZYWTXJDlG63XLqNdzQ)** using your **[evaluate.py](https://github.com/luckycallor/InsightFace-tensorflow/blob/master/evaluate.py)**, and the result is not as reported. Can you tell if there is something wrong with my setup?
  - script:
```
python evaluate.py --config_path configs/config_test.yaml --model_path config_ms1m_100_334k/best-m-334000
```
  - result:
```
749/750
done!
749/750
done!
done!
eval on agedb_30: acc--0.60483+-0.02297, tar--0.07433+-0.01248@far=0.00067
reading /home/thanhnn/dataset/faces_ms1m_112x112/lfw.bin
done!
forward running...
749/750
done!
749/750
done!
done!
eval on lfw: acc--0.93733+-0.02137, tar--0.76000+-0.04442@far=0.00133
reading /home/thanhnn/dataset/faces_ms1m_112x112/cfp_ff.bin
done!
forward running...
874/875
done!
874/875
done!
done!
eval on cfp_ff: acc--0.81229+-0.01246, tar--0.32200+-0.03002@far=0.00114
reading /home/thanhnn/dataset/faces_ms1m_112x112/cfp_fp.bin
done!
forward running...
874/875
done!
874/875
done!
done!
eval on cfp_fp: acc--0.68686+-0.01423, tar--0.14657+-0.02339@far=0.00086
done!
```

  - config_test.yaml
```
# model params
backbone_type: resnet_v2_m_50
loss_type: arcface
out_type: E

image_size: 112
embd_size: 512
class_num: 85742


# hyper params
bn_decay: 0.9
keep_prob: 0.4
weight_decay: !!float 5e-4
logits_scale: 64.0
logits_margin: 0.5
momentum: 0.9


# run params
val_bn_train: False
augment_flag: True
augment_margin: 16

gpu_num: 1
batch_size: 16
epoch_num: 20
step_per_epoch: 100000
val_freq: 2000

lr_steps: [40000, 60000, 80000]
lr_values: [0.004, 0.002, 0.0012, 0.0004]


# paths
# pretrained_model: '/data/hhd/InsightFace-tensorflow/output/20190120-133421/checkpoints/ckpt-m-140000'

train_data: ['/data/hhd/dataset/FaceData/InsightFace/faces_ms1m_arcface.tfrecord']
val_data: {'agedb_30': '/home/thanhnn/dataset/faces_ms1m_112x112/agedb_30.bin', 'lfw': '/home/thanhnn/dataset/faces_ms1m_112x112/lfw.bin', 'cfp_ff': '/home/thanhnn/dataset/faces_ms1m_112x112/cfp_ff.bin', 'cfp_fp': '/home/thanhnn/dataset/faces_ms1m_112x112/cfp_fp.bin'}

output_dir: './output'
```

**Comment by luckycallor** [comment]
@nnt296 
Thank you for proposing this issue.
I found there is a bug in evaluate.py, and it has been fixed. You can update your code and try again.
